File: model/ItemVenda.java
package model;

public class ItemVenda {
    private int idVenda;
    private int idProduto;
    private int qtd;  

    public ItemVenda(int idVenda, int idProduto, int qtd) {
        this.idVenda = idVenda;
        this.idProduto = idProduto;
        this.qtd = qtd;
    }
    public int getIdVenda() {
        return idVenda;
    }
    public void setIdVenda(int idVenda) {
        this.idVenda = idVenda;
    }
    public int getIdProduto() {
        return idProduto;
    }
    public void setIdProduto(int idProduto) {
        this.idProduto = idProduto;
    }
    public int getQtd() {
        return qtd;
    }
    public void setQtd(int qtd) {
        this.qtd = qtd;
    }
    @Override
    public String toString() {
        return "ItemVenda [idVenda=" + idVenda + ", idProduto=" + idProduto + ", qtd=" + qtd + "]";
    }
    
}

File: src/model/domain/ItemVenda.py
from model.database.BaseORM import Base
from sqlalchemy import Column, ForeignKey, Integer
from sqlalchemy.orm import relationship, backref
from model.domain.Venda import Venda
from model.domain.Produto import Produto

# class ItemVenda(Base):
    # __tablename__ = 'itemvenda'
    
    # qtd = Column(Integer, nullable=False)
    # id_venda = Column("idvenda", Integer, ForeignKey('venda.idvenda'), primary_key=True)
    # id_produto = Column("idproduto", Integer, ForeignKey('produto.idproduto'), primary_key=True)

    # venda = relationship("Venda")
    # #teste
    # #produto = relationship("Produto")
    # produto = relationship("Produto", back_populates="itemVenda")

class ItemVenda(Base):
    __tablename__ = 'itemvenda'
    
    qtd = Column(Integer, nullable=False)
    id_venda = Column("idvenda", Integer, ForeignKey('venda.idvenda'), primary_key=True)
    id_produto = Column("idproduto", Integer, ForeignKey('produto.idproduto'), primary_key=True)

    venda = relationship("Venda")
    produto = relationship("Produto", backref=backref("itemVenda"))